docs/content/latest/reference/configuration/yb-tserver.md

```diff
@@ -496,6 +496,14 @@ within `rocksdb_universal_compaction_size_ratio` of the next file in considerati
 
 Default: `20`
 
+##### --timestamp_history_retention_interval_sec
+
+The time interval in seconds to retain DocDB history for. Point-in-time reads at a hybrid time further than this in 
+the past might not be allowed after a compaction and return an `Snapshot too old` error. 
```

Comment:
```suggestion
might not be allowed after a compaction and return a `Snapshot too old` error. 
```

Reply:
Since it's number of seconds. Maybe we need `hybrid time further than these seconds in the past` ?